tests/visualization_tests/matplotlib_tests/test_contour.py

```diff
@@ -190,6 +202,7 @@ def test_plot_contour_log_scale_and_str_category() -> None:
     subplots = [plot for plot in figure.flatten() if plot.has_data()]
     expected = {"param_a": [1e-6, 1e-5], "param_b": [0.0, 1.0], "param_c": [0.0, 1.0]}
     ranges = itertools.permutations(expected.keys(), 2)
+    plt.savefig(BytesIO())
```

Comment:
This makes readability harder because this breaks the flow `range` is used in the following loop. 

[to end of the test case]